### ISSUE TITLE
--version flag being set to false even when providing value

### ISSUE THREAD

**Issue by jacksonopp** [jscutlery/semver#218]
Following the directions in the readme, I am attempting to update the versions for my affected projects.
`nx affected --target version --version=prerelease --preid=alpha`

The nx console then always shows that the `--version` flag has been set to `false` even though `prerelease` has been provided as a value.

```bash
>  NX  Running target version for 7 project(s):

  - component-tester
  - component-tester-e2e
  - documentation-e2e
  - documentation
  - navigation
  - scss-commons
  - ui-commons
  
  With flags:
    --version=false
    --preid=alpha
```

Running the `version` command on a specific project `nx run my-project:version [...options]` seems to work fine

**Comment by CaffeinatedCodeMonkey** [comment]
I'm having the same issue. Just some quick tinkering makes it look like it's butting up against the `--version` flag that exists for `nx affected ...`: https://nx.dev/latest/node/cli/affected#version.

Until it's fixed, it looks like I'll do some Bash magic and loop through the affected projects.

**Comment by mlefebvre-decathlon** [comment]
Hi :) 

I solved this issue by separating between the affected command options and the target options using an extra `--`

```
nx affected --target version -- --version=prerelease --preid=alpha
```

**Comment by CaffeinatedCodeMonkey** [comment]
> Hi :)
> 
> I solved this issue by separating between the affected command options and the target options using an extra `--`
> 
> ```
> nx affected --target version -- --version prerelease --preid alpha
> ```

Trying this actually causes an error for me: 
```
Error: Command failed: git merge-base --fork-point "affected" "--version"
error: unknown option `version'
```

Fortunately, what I'm doing right now works fine. I only want my apps to be versioned and the `affected` command doesn't seem to allow easy scoping to libs or apps while also using a custom target. So, I get a list of the candidates using `nx affected:apps --plain` (the `--plain` flag doesn't seem to be documented, but works nicely) to get a nice splittable list to iterate over and then call `nx run ${AFFECTED_PROJECT}:version --version=prerelease --preid=alpha` on each one.

**Comment by WonderPanda** [comment]
I'm also running into this when trying to use `nx affected` to version all packages that have been changed. Falling back to a script that uses `--plain` as recommended by @CaffeinatedCodeMonkey but it would be awesome if this worked out of the box

**Comment by edbzn** [comment]
Hi, this should be fixed in 2.8.1 using `--releaseAs=...` instead of `--version=...`.
